doc/cmpClient.pod

```diff
@@ -1191,11 +1221,59 @@ or L<openssl(1)/Verification Options> for details.
+=head1 HISTORY
+
+For an overview of CMP features relevant in industrial use cases see
+[LCMPP section 7.1](https://datatracker.ietf.org/doc/html/rfc9483#section-7.1).
+CMP client (EE) features are supported by the genCMPClient as follows.
+
+The features newly defined with CMPv3
+in [RFC 9480 (CMP Updates)](https://www.rfc-editor.org/rfc/rfc9480)
+are fully covered when using the [intermediate CMP library `libcmp`](
+https://github.com/mpeylo/cmpossl) or when using at least OpenSSL 3.5.
+
+* CMPv2 features defined in [RFC 4210](https://www.rfc-editor.org/rfc/rfc4210)
+  are sufficiently covered by using OpenSSL 3.0.\
+  This includes most of the
+  "Generic Aspects of PKI Messages and PKI Management Operations",
+  IR, CR, KUR, P10CR, MAC, RR, and polling for certification responses.
+* In OpenSSL 3.2, "Get CA Certificates" and "Get Root CA Certificate Update"
+  were added, as well as
+  the B<-srvcertout> option for saving the validated server certificate and
+  the B<-serial> option for the serial number of certificates to be revoked.
+  Since then, the B<-issuer> may be used also for certificates to be revoked.
+* In OpenSSL 3.3, support for certificate profiles with the B<-profile> option
+  and generalized polling ("Handling Delayed Delivery") were added, as well as
+  the B<-no_cache_extracerts> option for not reusing extraCerts received.
+* In OpenSSL 3.4, "Get Certificate Request Template" and
+  "CRL Update Retrieval" were added, as well as the related B<-template>,
+  B<-crlcert>, B<-oldcrl>, and B<-crlout> options.
+* In OpenSSL 3.5, support for central key generation is added.
+
+
 =head1 COPYRIGHT
 
-Copyright (c) 2021 Siemens AG.
+Copyright (c) 2021-2024 Siemens AG.
 
 Licensed under the Apache License, Version 2.0
 SPDX-License-Identifier: Apache-2.0
 
 =cut
+
+#  LocalWords:  cmpClient cli md doc pkcs tls cert cmd ir cr kur rr
+#  LocalWords:  genm infotype geninfo newkeytype len centralkeygen cn
+#  LocalWords:  newkey newkeypass arg reqexts san nodefault oids popo
+#  LocalWords:  csr ip certout chainout oldcert revreason msg srvcert
+#  LocalWords:  keyusage extracerts srvcertout extracertsout dir ref
+#  LocalWords:  dirname cacertsout cacerts oldwithold newwithnew mac
+#  LocalWords:  newwithold oldwithnew oldcrl crlout keypass reqin tid
+#  LocalWords:  reqout rspin rspout crls cdp cdps url crl maxdownload
+#  LocalWords:  aia ocsp num auth attime ss sig suiteB alt certs CRIT
+#  LocalWords:  EMERG nbsp stdout signKeyPairTypes caCerts rootCaCert
+#  LocalWords:  certReqTemplate crlStatusList int str genp keySpec fd
+#  LocalWords:  env DC org RAVERIFIED KEYENC crlcert CRLReason RP
+#  LocalWords:  keyCompromise cACompromise affiliationChanged PKIConf
+#  LocalWords:  cessationOfOperation certificateHold removeFromCRL CP
+#  LocalWords:  privilegeWithdrawn aACompromise PKIStatus pollRep crt
+#  LocalWords:  ErrMsgContent pem KUP thisUpdate subjectName ALG sha
+#  LocalWords:  openssl hmacWithSHA hmac recipNonce CMPv libcmp
```

Comment:
Are these required? look like wrongly pasted.

Reply:
This was added by `ispell`, e.g., in Emacs, to hold local dictionary values.
At least for me as the main author of this text, this is helpful to have.